test/hotspot/jtreg/TEST.groups

```diff
@@ -444,6 +444,19 @@ tier1 = \
   :tier1_runtime \
   :tier1_serviceability
 
+tier2 = \
+  -:tier1 \
+  :hotspot_tier2_runtime \
+  :hotspot_tier2_serviceability \
+  :tier2_gc_shenandoah
+
+tier3 = \
+  -:tier1 \
+  -:tier2 \
+  :hotspot_tier3_runtime \
+  -runtime/InvocationTests \
```

Comment:
Why are these excluded?

Reply:
Sorry I missed the comment in the PR.

What is your time criteria?

Reply:
So the current `runtime/InvocationTests` run for about 35 minutes real and 58 minutes user time. This blows up tier3 real time for about 5x. So I chickened out adding them to tier3. But after sleeping on it, I think those still make sense to run in tier3, and then we should optimize the tests themselves, for example with [JDK-8272970](https://bugs.openjdk.java.net/browse/JDK-8272970). See new commit.